test/unit/index.test.js

```diff
@@ -18,6 +22,15 @@ describe('sails-hook-cron', () => {
           start: false,
           timezone: 'Europe/Kiev',
           context: undefined
+        },
+        sailsJob: {
+          schedule: '* * * * * *',
+          onTick: function() {
+            console.log("Executing onTick from Sails Cron Job");
+            this.config.sailsWorks = true;
+          },
+          onComplete: console.log,
```

Comment:
The same, there is no reason to make calls to the console, it will make stdout dirty.